WindowsServerDocs/identity/ad-fs/operations/AD-FS-Rapid-Restore-Tool.md

```diff
@@ -57,7 +57,7 @@ import-module 'C:\Program Files (x86)\ADFS Rapid Recreation Tool\ADFSRapidRecrea
 ### System requirements
 
 - This tool works for AD FS in Windows Server 2012 R2 and later.
-- The required .NET framework is at least 4.0.
+- The required .NET framework is at least 4.0. (Note: For version 1.0.82.3 or later, it requires at least .NET framework 4.6)

```

Comment:
For consistency, I recommend using the MarkDown Note blob format used elsewhere in this document:
```suggestion
- The required .NET framework is at least 4.0.
  > [!NOTE]
  > For version 1.0.82.3 or later, at least .NET Framework 4.6 is required.
```
(I have also added the suggestion to use capitalization of part of the trademark name "Framework".)